### ISSUE TITLE
java.lang.String cannot be cast to com.facebook.react.uimanager.AccessibilityDelegateUtil$AccessibilityRole - on 0.57.3

### ISSUE THREAD

**Issue by varunajmera0** [facebook/react-native#21816]
![screen shot 2018-10-16 at 10 45 23 pm](https://user-images.githubusercontent.com/22893376/47034608-823a6f00-d195-11e8-8f12-c3e6274087bd.png)

**Comment by radko93** [comment]
@facebook-github-bot needs-repro

**Comment by radko93** [comment]
@facebook-github-bot no-template
